src/k8s/hack/gen_cmd_docs.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"log"
+
+	"github.com/canonical/k8s/cmd/k8s"
+	"github.com/spf13/cobra/doc"
+)
+
+func main() {
+	k8sCmd := k8s.NewRootCmd()
+
+	err := doc.GenMarkdownTree(k8sCmd, "../../../docs/src/_parts/commands")
+	if err != nil {
+		log.Fatal(err)
+	}
+}
```

Comment:
could we make this a hidden `k8s documentation` command? That way you can avoid the `-C ${DIR}` things below

Reply:
i can make this a `k8s $cmd` and generate docs at runtime, but i'm not sure how to call it from a Makefile / gh action that can determine if a PR-under-test is missing docs.

Reply:
aha!  call it like you would any other command, so long as it's built/linked in the go.doc target.  thanks @neoaggelos!